### PR TITLE
fix: reset stickiness to default on variants modal close

### PR DESCRIPTION
**Pull request by andreas-unleash**
Expanded the onClose handler to reset stickiness to default for project on modal close
## About the changes

Closes #

### CODE REVIEW

File: frontend/cypress/integration/projects/settings.spec.ts [hunk 1]
```diff
@@ -68,6 +66,7 @@ describe('project settings', () => {
         );
 
         cy.get("[data-testid='ADD_VARIANT_BUTTON']").first().click();
+        cy.wait(300);
```
Comment (kwasniew) [inline frontend/cypress/integration/projects/settings.spec.ts:69]: I'd rather await on the element that appears asynchronously. 300ms wait is very arbitrary.

File: frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/EnvironmentVariantsModal/EnvironmentVariantsModal.tsx [hunk 2]
```diff
@@ -307,18 +307,17 @@ export const EnvironmentVariantsModal = ({
         }
     }, [apiPayload.error]);
 
+    const handleClose = () => {
+        updateStickiness(defaultStickiness).catch(console.warn);
```
Comment (kwasniew) [inline frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/EnvironmentVariantsModal/EnvironmentVariantsModal.tsx:311]: do we want this console.warn in the catch block?

**Review by kwasniew — APPROVED** [review]
non blocking approve but I'd not add artificial wait time in the tests

**Comment by andreas-unleash** [comment]
removed the wait time

**Pull request merged**